src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -40,7 +40,8 @@ public class ObjectModuleDetails
             this.CompilerBackEndVersion = backEndVersion ?? new Version();
 
             // The command line for link.exe should contain /OUT with the final binary name.  Use that to separate compiler and linker command lines.
-            if ((commandLine != null) && commandLine.Contains("/OUT"))
+            if ((commandLine != null) &&
+                (commandLine.Contains("/OUT", System.StringComparison.OrdinalIgnoreCase) || commandLine.Contains("-OUT", System.StringComparison.OrdinalIgnoreCase)))
```

Comment:
>commandLine [](http://example.com/codeflow?start=17&length=11)

Good fix. Since you put the hood up here, can we please create a small utility method that performs this case-insensitive, prefix-insensitive ('-' vs. '/') and then consume it here and elsewhere? Didn't want to block your previous PR where the same thought occurred to me. It'd be a good small clean-up change, I think.

Reply:
Got a recommendation on where such a helper should go?  I don't see an obvious `Utils.cs` to stash it.

Reply:
Since it is for the linker command-line in both cases it could arguably be a static method on `LinkerCommandLine`.  The OM code calls in and says "Is this a linker command line?" and gets a yes/no response.  Internally it uses the same helper for the other linker options.

Reply:
I went ahead with wrapping it up into LinkerCommandLine because it is currently specific to that logic.  There are comparisons with `Contains`, `StartsWith`, and `Equals` so it is a little bit messy to have several helpers.

I also needed to substring into the target to exclude the prefix which will generate some garbage.  There should only be one linker command line per binary so the impact of that should be minimal.